src/patterns/problem-with-the-service-pages/index.md

```diff
@@ -49,6 +51,21 @@ Have clear and concise content and do not use:
 - ‘We are experiencing technical difficulties’
 - red text to warn people
 
+### Showing contact information
+
+Contact information should either:
+
+- include all numbers and opening times in one place on the page
+- be a link out to a page that includes numbers and opening times
+
+You might choose to link to a ‘contact information’ page, such as one shown in the ‘[Contact a department or service team](https://design-system.service.gov.uk/patterns/contact-a-department-or-service-team/)’ pattern.
+
+<strong class="govuk-tag govuk-tag--grey">WCAG 2.2</strong> You must always write contact information in a clear and consistent way across ‘There is a problem with the service’ pages. This relates to WCAG 2.2 success criterion [3.2.6 Consistent Help](https://www.w3.org/WAI/WCAG22/Understanding/consistent-help.html).
+
+Whenever possible, make sure to save and store any information the user has previously entered, unless doing so would be a major safety or security concern. This is to comply with WCAG 2.2 success criterion [3.3.7 Redundant Entry](https://www.w3.org/WAI/WCAG22/Understanding/redundant-entry.html).
+
+Store previously entered information for a reasonable amount of time so users can resume a journey with re-populated information when the service becomes available again.
+
```

Comment:
Now that I've looked at it, it might be better to split these 2 messages in this instance.

```suggestion
### Showing contact information

Contact information should either:

- include all numbers and opening times in one place on the page
- be a link out to a page that includes numbers and opening times

You might choose to link to a ‘contact information’ page, such as one shown in the ‘[Contact a department or service team](https://design-system.service.gov.uk/patterns/contact-a-department-or-service-team/)’ pattern.

<strong class="govuk-tag govuk-tag--grey">WCAG 2.2</strong> You must always write contact information in a clear and consistent way across ‘There is a problem with the service’ pages. This relates to WCAG 2.2 success criterion [3.2.6 Consistent Help](https://www.w3.org/WAI/WCAG22/Understanding/consistent-help.html).

Store previously entered information for a reasonable amount of time so users can resume a journey with re-populated information when the service becomes available again.

<strong class="govuk-tag govuk-tag--grey">WCAG 2.2</strong> Whenever possible, make sure to save and store any information the user has previously entered, unless doing so would be a major safety or security concern. This is to comply with WCAG 2.2 success criterion [3.3.7 Redundant Entry](https://www.w3.org/WAI/WCAG22/Understanding/redundant-entry.html).

```